parsec/remote_dep.h

```diff
@@ -29,33 +38,30 @@ typedef ptrdiff_t remote_dep_datakey_t;
 #define PARSEC_ACTION_RESHAPE_REMOTE_ON_RELEASE  0x80000000
 #define PARSEC_ACTION_RELEASE_REMOTE_DEPS        (PARSEC_ACTION_SEND_INIT_REMOTE_DEPS | PARSEC_ACTION_SEND_REMOTE_DEPS)
 
-typedef struct remote_dep_wire_activate_s
-{
+enum {
```

Comment:
Missing `typedef` in front of `enum`. Surprisingly, this does not create an error unless you try to link the Fortran files (in which case the `parsec_remote_dep_tag_t` /variable/ is multiply-defined).